### PR TITLE
[StyleCop] Fix warnings in TimeAssemblyInfo

### PR DESCRIPTION
**Pull request by Bill7zz**
-Remove trailing whitespace
-Rename variable with Hungarian notation

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/TimexCreator.cs [hunk 1]
```diff
@@ -33,6 +33,7 @@ public static string Tomorrow(DateObject date = default(DateObject))
             d = d.AddDays(1);
             return TimexProperty.FromDate(d).TimexValue;
```
Comment (VictorGrycuk) [inline .NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/TimexCreator.cs:34]: Please add a new line before the `return`.

File: .NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/TimexCreator.cs [hunk 2]
```diff
@@ -86,7 +87,7 @@ public static string LastWeek(DateObject date = default(DateObject))
             return t.TimexValue;
```
Comment (VictorGrycuk) [inline .NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/TimexCreator.cs:87]: Add a new line before the `return`.

**Review by VictorGrycuk — DISMISSED** [review]
In your PR details, add a space after the hyphen to properly format the bullets.

Disregard previous requested changes as they were not part of CodeStyle.

**Review by VictorGrycuk — APPROVED** [review]
LGTM!

**Review by JuanAr — APPROVED** [review]
LGTM

**Review by ceciliaavila — COMMENTED** [review]
LGTM

**Pull request closed without merging**